package/piksi_leds/src/manage_led.c

```diff
@@ -214,6 +218,10 @@ static void handle_link(rgb_led_state_t *s) {
     on_off = false;
   }
 
+  if (network_available && (*network_available == '1')) {
+    on_off = !on_off;
```

Comment:
I don't think this implements the behavior correctly, we're supposed to blink if we have obs + network, doing this here will just flip back whatever on_off was changed to by the obs check.

Reply:
This inverts it.  obs with no network is normally off with short blinks on, obs with network is normally on with short blinks off.

Reply:
From the design doc:
> If red LED is solid on and blinks off intermittently, Duro is connected to the internet and is receiving observations from Skylark / a CORS"